mtc.go

```diff
@@ -441,6 +446,86 @@ func (c *BikeshedCertificate) UnmarshalBinary(data []byte, caStore CAStore) erro
 	return nil
 }
 
+// VerifyOptions includes parameters for verifying a BikeshedCertificate.
+type VerifyOptions struct {
+	// ValidityWindow is a validity window that covers the certificate. It
+	// is the caller's responsibility to verify the validity window was
+	// signed by the CA, e.g., by verifying the SignedValidityWindow that
+	// contains the validity window.
+	ValidityWindow *ValidityWindow
+
+	// CA includes the parameters of the CA that issued the batch
+	// containing the certificate.
+	CA *CAParams
+
+	// CurrentTime is used to to check if the certificate has expired.
+	CurrentTime time.Time
+}
+
+// Verify is used to verify that a BikeshedCertificate is covered by a validity
+// window. It is the caller's responsibility to verify the validity window was
+// signed by the CA. An error indicates that the certificate does not belong to
+// a batch in the validity window or that the certificate is otherwise invalid.
+func (c *BikeshedCertificate) Verify(opts VerifyOptions) error {
+	if opts.ValidityWindow == nil {
+		return errors.New("Missing validity window")
+	}
+
+	if opts.CA == nil {
+		return errors.New("Missing CA parameters")
+	}
+
+	p, ok := c.Proof.(*MerkleTreeProof)
+	if !ok {
+		return fmt.Errorf("Expected MerkleTreeProof; got %t", c.Proof)
+	}
+	certBatchNumber := p.TrustAnchorIdentifier().BatchNumber
+
+	// If the current time wasn't set, then use use time.Now().
+	currentTime := opts.CurrentTime
+	if currentTime.IsZero() {
+		currentTime = time.Now()
+	}
+
+	// Check if the certificate is not yet valid.
+	notBefore := time.Unix(int64(opts.CA.StartTime), 0).Add(
+		time.Duration(uint64(certBatchNumber)*opts.CA.BatchDuration) *
+			time.Second)
```

Comment:
You can use Batch.ValidityInterval.

Reply:
Done!  While at it, I improved the documentation by noting that the NotAfter might be smaller than the end of the interval.